File: app/src/main/java/id/ac/unej/inaparadise/fragment/TabPermainan.java
package id.ac.unej.inaparadise.fragment;

import android.content.Intent;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.AdapterView;
import android.widget.GridView;

import id.ac.unej.inaparadise.R;
import id.ac.unej.inaparadise.adapter.GridViewAdapter;
import id.ac.unej.inaparadise.ui.MapActivity;

public class TabPermainan extends Fragment {

    public TabPermainan() {

    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View root = inflater.inflate(R.layout.fragment_tab_permainan, container, false);
        GridView gridview = root.findViewById(R.id.menu_kebudayaan);
        gridview.setAdapter(new GridViewAdapter(getActivity()));

        gridview.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            public void onItemClick(AdapterView<?> parent, View v, int position, long id) {
//                Toast.makeText(HelloGridView.this, "" + position, Toast.LENGTH_SHORT).show();
                startActivity(new Intent(getActivity(), MapActivity.class));
            }
        });

        return root;
    }
}

File: app/src/main/java/id/ac/unej/inaparadise/fragment/TabPermainan.kt
package id.ac.unej.inaparadise.fragment

import android.support.v4.app.Fragment
import android.view.View
import id.ac.unej.inaparadise.ui.MapActivity

class TabPermainan : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val root: View = inflater.inflate(R.layout.fragment_tab_permainan, container, false)
        val gridview: GridView = root.findViewById<GridView>(R.id.menu_kebudayaan)
        gridview.setAdapter(GridViewAdapter(activity))
        gridview.setOnItemClickListener(object : AdapterView.OnItemClickListener {
            override fun onItemClick(parent: AdapterView<*>?, v: View, position: Int, id: Long) {
//                Toast.makeText(HelloGridView.this, "" + position, Toast.LENGTH_SHORT).show();
                startActivity(Intent(activity, MapActivity::class.java))
            }
        })
        return root
    }
}